websites/D/DscJobs/presence.ts

```diff
@@ -0,0 +1,59 @@
+const presence = new Presence({
+  clientId: "807748912940711996"
+}),
+time = Math.floor(Date.now() / 1000),
+path = window.location.pathname;
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo",
+    startTimestamp: time
+  };
+
+  switch (path) {
+    case '/': presenceData.details = "Viewing Home Page"; break;
+    case '/premium': presenceData.details = "Viewing Premium Page"; break;
+    case '/create': presenceData.details = "Creating Profile"; break;
+    case '/legal': presenceData.details = "Viewing Legal Page"; break;
+    case '/partners': presenceData.details = "Viewing Partners Page"; break;
+    case '/supporters': presenceData.details = "Viewing Supporters Page"; break;
+    case '/moderators': presenceData.details = `Viewing Moderators Page ${location.href.split("page=")[1].split("&")[0]}`; break;
```

Comment:
I don't think it will look good when it will be like: `Viewing Moderators Page 1`

Reply:
I changed it

Reply:
only this and it will be approved?